### PR TITLE
Improve ReadMe

### LINKED ISSUE #5: Update readme
I had a bit of fiddling to get it going:
- There's a ruby version file, which meant my version manager made me install it!
- We don't need to install rails manually. Rails is a gem, so once you've done `rails new` it'll live in the gemfile and we'll only need to do `bundle install`
- There is a migration already! Means we'll need to run `rails db:create`, `rails db:migrate`

### PR DESCRIPTION
**Pull request by Emmett-42**
To tailor the ReadMe file to provide the required specific information to reach and run the application and serve the purpose of a ReadMe file the best.

Resolves https://github.com/Ayesha-42/HangmanRails/issues/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# README
+# Hangman Web Application Game
 
-This is a simple web application game of Hangman
+This is a simple web application game of Hangman.
 
 In order to run this game on your local system, keep these specs and steps in mind:
 
-* Ruby versions 2.7.0 or later to run Rails
```
Comment (MizukiZ) [inline README.md:7]: `bundle install` install all the dependencies (including rails), and that manage the specific versions of gems. So here you can say do `bundle install`

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+* Ruby versions 2.7.0 or later is required to support Rails.
 
-* Use ```gem install rails``` to install rails 7.0.0 or later
+* Use ```gem install rails``` to install rails 7.0.0 or later, if not previously installed.
 
-* Use ```bundle install``` to run the Gemfile and install/update all the required gems from the app directory ```cd HangmanRailsGame```
+* From the HangmanRailsGame directory use ```bundle install``` to run the Gemfile and install/use all the required gems.
 
-* To start the web server to run your web application use ```rails server``` command.
```
Comment (MizukiZ) [inline README.md:13]: `rails server` is correct but we also can do `rails s` instead :)

**Comment by MizukiZ** [comment]
Look good Ayasha, left a couple of comments.
You can make readme look a bit nicer using code block https://github.com/JetBrains/sample_rails_app

For example,
```
bundle install
```
instead of
```bundle install```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-This is a simple web application game of Hangman
+This is a simple web application game of Hangman.
 
 In order to run this game on your local system, keep these specs and steps in mind:
 
-* Ruby versions 2.7.0 or later to run Rails
+* Ruby versions 2.7.0 or later is required to support Rails.
 
-* Use ```gem install rails``` to install rails 7.0.0 or later
+* Use ```gem install rails``` to install rails 7.0.0 or later, if not previously installed.
```
Comment (MizukiZ) [inline README.md]: `bundle install` installs all the dependencies (including rails), and that manages the specific versions of gems.  Therefore, this step is not necessary I think.

File: README.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 
-* Use ```bundle install``` to run the Gemfile and install/update all the required gems from the app directory ```cd HangmanRailsGame```
+* From the HangmanRailsGame directory use ```bundle install``` to run the Gemfile and install/use all the required gems.
 
-* To start the web server to run your web application use ```rails server``` command.
+* To start the web server to run your web application use ```rails server```.
```
Comment (MizukiZ) [inline README.md]: `rails server` is correct but we also can do `rails s` instead :)

Comment (Emmett-42) [reply README.md]: Thanks for the inputs, will make the necessary changes.

**Review by MizukiZ — APPROVED** [review]
Looks great thanks for the updates

**Pull request merged**